### PR TITLE
NSIS syntax update

### PR DESCRIPTION
**Pull request by sredna**
Instruction list auto-generated directly from NSIS source. This means some of the v1.0 instructions you have in your list (but we (NSIS) removed 20 years ago) are no longer recognized.

### CODE REVIEW

File: PowerEditor/installer/APIs/nsis.xml [hunk 1]
```diff
@@ -98,22 +107,27 @@
 		<KeyWord name="GetDLLVersion" />
 		<KeyWord name="GetDLLVersionLocal" />
 		<KeyWord name="GetErrorLevel" />
-		<KeyWord name="GetExeName" />
-		<KeyWord name="GetExePath" />
```
Comment (donho) [inline PowerEditor/installer/APIs/nsis.xml:102]: Why are both APIs removed?

File: PowerEditor/installer/APIs/nsis.xml [hunk 2]
```diff
@@ -203,7 +236,6 @@
 		<KeyWord name="SetRegView" />
 		<KeyWord name="SetShellVarContext" />
 		<KeyWord name="SetSilent" />
-		<KeyWord name="SetStaticBkColor" />
```
Comment (donho) [inline PowerEditor/installer/APIs/nsis.xml:206]: It seems a valid API too.

File: PowerEditor/installer/APIs/nsis.xml (hunk 1 above)
Comment (sredna) [reply PowerEditor/installer/APIs/nsis.xml:102]: They do not even exist in the first CVS2SVN check-in 20 years ago: https://sourceforge.net/p/nsis/code/625/tree/NSIS/trunk/Source/tokens.cpp There are some macros with those names but the syntax would be ${GetExeName} and they are not a part of the base instructions.

$EXEPATH and $EXEFILE variables have been around for 15 years.

File: PowerEditor/installer/APIs/nsis.xml (hunk 2 above)
Comment (sredna) [reply PowerEditor/installer/APIs/nsis.xml:206]: Replaced in 2003: https://sourceforge.net/p/nsis/code/2347/

**Pull request closed without merging**